graftm/hmmer.py

```diff
@@ -318,10 +353,10 @@ def __check_euk_contamination(self, hmm_hit_tables, run_stats):
         
         Returns
         -------
-        run_stats : dict
-            Updated form of the above.
+        euk_reads : array
```

Comment:
I just mean the line in the documentation. The fact that the variable returned is named `euk_reads` is not relevant outside this function. I would just say
```
Returns
-------
array of all reads names deemed to be eukaryotic
```